CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Note: CMake support is community-based. The maintainers do not use CMake
 # internally.
 
-cmake_minimum_required(VERSION 2.8.8)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
```cmake
# Adding ... to version rage to future proof code when using cmake > 3.18 that explicitly 
# deprecated 2.8.12 features
cmake_minimum_required(VERSION 2.8.12...)
```

Reply:
https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html

Reply:
Version range was added in 3.12 to this command and the goal is to support 2.8.12 as a minimum.

Also, empty max range?

Reply:
From the linked docs:

> The optional <max> version, if specified, must be at least the <min> version and affects policy settings as described below. If the running version of CMake is older than 3.12, the extra ... dots will be seen as version component separators, resulting in the ...<max> part being ignored and preserving the pre-3.12 behavior of basing policies on <min>.